app/serializers/v1/task_serializer.rb

```diff
@@ -0,0 +1,7 @@
+module V1
+  class TaskSerializer < ActiveModel::Serializer
+
+    attributes :id, :name, :description, :state
```

Comment:
I left out the project_id because the serializer is used when listing tasks from a project, so the project id is required to even get the list, and if we were using a serializer for it in the first place I saw no reason to have the project_id included.  If it were to be used in the future for listing all tasks, a different serialization for it that included the project_id would make more sense to me.